less/admin/AdminNav.less

```diff
@@ -1,17 +1,83 @@
-@admin-pane-width: 300px;
+@admin-pane-width: 210px;
```

Comment:
I think this is too tight, 300 seems to actually be the perfect size, but if we could at least go above 250, otherwise 210 is really tight.

Reply:
Agreed, some extensions with longer names are cut off.